src/test/java/org/jabref/logic/bibtex/BibDatabaseMergerTest.java

```diff
@@ -0,0 +1,134 @@
+package org.jabref.logic.bibtex;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibtexString;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.types.StandardEntryType;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class BibDatabaseMergerTest {
+
+    @Test
+    void mergeAddsNonDuplicateEntries() {
+        // Entries 2 and 3 are identical
+        BibEntry entry1 = new BibEntry()
+                .withField(StandardField.AUTHOR, "Stephen Blaha")
```

Comment:
Please reduce these examples to a minimum. I don't think you need to have 4 entries with full information

Reply:
I reduced the number of entries and their information content.